cmd/nodeagent/nodeagentmanager_test.go

```diff
@@ -809,7 +809,7 @@ var _ = Describe("nodeagent tests", func() {
 		// change time to be 6 seconds later again
```

Comment:
should we change this comment as well?

Reply:
No it's okay, the previous time was 10:00:06, so i wasn't actually changing the time again.